api/client/cli.go

```diff
@@ -93,10 +93,11 @@ func (cli *DockerCli) Subcmd(name, signature, description string, exitOnError bo
 	flags := flag.NewFlagSet(name, errorHandling)
 	flags.Usage = func() {
 		options := ""
+		tempSignature := " " + signature
```

Comment:
Yes, maybe I have make a confuse for myself.
Use a spaces before options should be much better than trailing space, because trailing space should make some confuse for signature.
OK, anyway, I will commit the change.thanks.